tools/clang/test/HLSLFileCheck/passes/llvm/instcombine/sink.hlsl

```diff
@@ -0,0 +1,38 @@
+// RUN: %dxc %s /T ps_6_0 -opt-disable sink | FileCheck %s
```

Comment:
I think you have the sink.hlsl and nosink.hlsl test cases reversed (i.e. sink should be enabled here based on the file name).

Reply:
Fixed.